scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+            locations = [
+                loc for loc in all_locations 
+                if loc.metadata.get('type') == 'city_center'
+            ][:10]  # Top 10 cities
+        
+        logger.info(f"Downloading data for {len(locations)} locations")
+        logger.info(f"Parameters: {parameters}")
+        logger.info(f"Period: {start_date.date()} to {end_date.date()}")
+        logger.info(f"Data type: {'Forecast' if use_forecast else 'Reanalysis'}")
+        
+        # Create output directory
+        output_dir = output_dir or Path(f"data/cams/processed")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)